docs/source/notes/autograd.rst

```diff
@@ -210,3 +210,76 @@ No thread safety on C++ hooks
 Autograd relies on the user to write thread safe C++ hooks. If you want the hook
 to be correctly applied in multithreading environment, you will need to write
 proper thread locking code to ensure the hooks are thread safe.
+
+Vector-Jacobian Product and Jacobian-Vector Product for complex numbers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Consider a function F: V → W, where are V and W are vector spaces. As described in
+Chapter 4 of Dougal's thesis, the output of Jacobian-Vector Product (JVP) function
```

Comment:
Usually best to have a link or a footnote here

Reply:
Maybe this is fine for now, but if I were trying to have this doc be as readable as possible, there would probably need to be an introduction of JVP/VJP somewhere here (ordinarily I'd expect it to be explained earlier, but no dice in the existing doc).